PayPal.xcodeproj/xcshareddata/xcschemes/UnitTests.xcscheme

```diff
@@ -34,13 +34,28 @@
                ReferencedContainer = "container:PayPal.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "NO"
+            buildForProfiling = "NO"
+            buildForArchiving = "NO"
+            buildForAnalyzing = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "PayPal::PayPalTests"
+               BuildableName = "PayPalTests.xctest"
+               BlueprintName = "PayPalTests"
+               ReferencedContainer = "container:PayPal.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
```

Comment:
This was also automatically added -there were already entries here for `PaymentsCoreTests` and `CardTests`, so not sure why `PayPalTests` was treated differently until now.